configs/default/SPRO-SPRACINGH7EXTREME.config

```diff
@@ -107,32 +101,35 @@
 #define ADC1_DMA_OPT 8
 #define ADC3_DMA_OPT 9
 
+#ifndef USE_FLASHFS
 #define USE_FLASHFS
+#endif
 #define ENABLE_BLACKBOX_LOGGING_ON_SDCARD_BY_DEFAULT
 
 #
 # Settings that are currently defined in target/common_pre.h for non-cloud builds that probably shouldn't be.
 # There are here to illustrate that they SHOULD be included in THIS target when they are removed by default.
 #
 
-#define _USE_MAG
-#define _USE_MAG_HMC5883
-#define _USE_MAG_QMC5883
-#define _USE_BARO
-#define _USE_BARO_BMP388
-#define _USE_TRANSPONDER
-#define _USE_OSD
-#define _USE_MAX7456
-#define _USE_VTX_COMMON
-#define _USE_VTX_CONTROL
-#define _USE_VTX_RTC6705_SOFTSPI
+#define USE_ACC_SPI_ICM20602
+#define USE_GYRO_SPI_ICM20602
+#define USE_BARO_BMP388
+#define USE_MAG_HMC5883
+#define USE_MAG_QMC5883
+#define USE_FLASH_W25N01G
+#define FLASH_QUADSPI_INSTANCE    QUADSPI
+#define CONFIG_IN_EXTERNAL_FLASH
+#define USE_SDCARD_SDIO
+#define USE_CAMERA_CONTROL
+#define USE_MAX7456
+#if defined(USE_VTX) && !defined(USE_VTX_RTC6705_SOFTSPI)
```

Comment:
#if def will not work. The flags are either in or not. This will need to be moved to some other trigger in common

Reply:
Removed as it's added for H7 by default